### PR TITLE
add acf image_aspect_ratio_crop as related item (WP-902)

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
This appears to be a rarely used acf field type. The client should now be able to send related images from the related UI.

### CODE REVIEW

**Review by dimitrystd — APPROVED** [review]

**Review by PavelLoparev — APPROVED** [review]

File: inc/Smartling/Tuner/CustomizationManagerAbstract.php [hunk 1]
```diff
@@ -11,7 +11,11 @@ abstract class CustomizationManagerAbstract implements ManagerInterface, \Iterat
 
     protected function generateId(int $length = 12): string
     {
-        return substr(md5(microtime(true)), 0, $length);
+        if ($length > 23) {
+            throw new \RuntimeException("Length should be less than 23 characters");
```
Comment (PavelLoparev) [inline inc/Smartling/Tuner/CustomizationManagerAbstract.php:15]: `Less than or equal to` according to the condition

File: inc/config/media-attachment-rules.yml [hunk 2]
```diff
@@ -33,3 +35,17 @@ services:
...
+    class: Smartling\Helpers\GutenbergReplacementRule
+    arguments:
+      - "acf/.+"
+      - "name"
+      - "copy"
+
+  wordpress.editor.mode:
+    class: Smartling\Helpers\GutenbergReplacementRule
+    arguments:
+      - ".+"
```
Comment (dimitrystd) [inline inc/config/media-attachment-rules.yml:49]: This mean "everything". I don't know what GutenbergReplacementRule does, but want to draw your attention.

**Review by dimitrystd — APPROVED** [review]

File: tests/IntegrationTests/testdata/wp-690-expected.html [hunk 3]
```diff
@@ -26,7 +26,7 @@ <h1 class="uk-margin-remove-top">[D~ólór]</h1>
...
+<!-- wp:acf/inner-tabs {"id":"[b~lóc~k_61~9229~8144~55é]","name":"acf/inner-tabs","data":{"style":"[ú~ñdér~líñé]","_style":"field_6154aa83b71bb","tabs_0_name":"[L~óré~m Íp~súm]","_tabs_0_name":"field_602b9e5859da4","tabs_0_anchor":"[#~lóré~m-íp~súm]","_tabs_0_anchor":"field_602b9e6659da5","tabs_1_name":"[Ú~sé Cá~sés]","_tabs_1_name":"field_602b9e5859da4","tabs_1_anchor":"[#~úsé~-cá~sés]","_tabs_1_anchor":"field_602b9e6659da5","tabs_2_name":"[V~ídéó ~Démó]","_tabs_2_name":"field_602b9e5859da4","tabs_2_anchor":"[#~víd~éó-d~émó]","_tabs_2_anchor":"field_602b9e6659da5","tabs_3_name":"[R~élát~éd Ré~sóúr~cés]","_tabs_3_name":"field_602b9e5859da4","tabs_3_anchor":"[#~rél~átéd~-ré~sóúr~cés]","_tabs_3_anchor":"field_602b9e6659da5","tabs_4_name":"[G~ét S~tár~téd]","_tabs_4_name":"field_602b9e5859da4","tabs_4_anchor":"[#~gét-~stár~téd]","_tabs_4_anchor":"field_602b9e6659da5","tabs":5,"_tabs":"field_602b9e4159da3"},"align":"","mode":"edit"} /-->
```
Comment (PavelLoparev) [inline tests/IntegrationTests/testdata/wp-690-expected.html:29]: 👍

Comment (PavelLoparev) [reply tests/IntegrationTests/testdata/wp-690-expected.html:29]: `"id":"[b~lóc~k_61~9229~8144~55é]"` - is this also ok?

**Review by PavelLoparev — APPROVED** [review]

File: inc/Smartling/Tuner/CustomizationManagerAbstract.php (hunk 1 above)
Comment (vsolovei-smartling) [reply inc/Smartling/Tuner/CustomizationManagerAbstract.php]: Fixed

File: inc/config/media-attachment-rules.yml (hunk 2 above)
Comment (vsolovei-smartling) [reply inc/config/media-attachment-rules.yml]: You are correct, restricted to acf/.+

File: tests/IntegrationTests/testdata/wp-690-expected.html (hunk 3 above)
Comment (vsolovei-smartling) [reply tests/IntegrationTests/testdata/wp-690-expected.html:29]: This is block-dependent, should not be a hardcoded rule

File: inc/config/media-attachment-rules.yml (hunk 2 above)
Comment (PavelLoparev) [reply inc/config/media-attachment-rules.yml]: I think you need to escape `/`

Comment (vsolovei-smartling) [reply inc/config/media-attachment-rules.yml]: Checked, the escape is not needed

**Pull request merged**